libvmi/os/windows/windows.h

```diff
@@ -23,6 +23,10 @@
 
 #include "private.h"
 
+#define LOWEST_KERNEL_ADDRESS_64BIT 0xffff800000000000
+#define LOWEST_KERNEL_ADDRESS_32BIT 0x80000000
```

Comment:
Acutally I don`t think it is needed at all.
This is used to sanity check the kpcr ( [Issue #354](https://github.com/libvmi/libvmi/issues/354) ) There should be no problem with the kpcr in the case of VMI_PM_LEGACY or VMI_PM_PAE (in contrast to VMI_PM_IA32E where a wrong kpcr is read if the currently running process is a wow64 process).